docs/docker_usage.md

```diff
@@ -105,5 +94,11 @@ Now, supposing that the configuration file is located at
 docker run \
 -v ~/Desktop/cnwan-credentials/serv-acc.json:/credentials/serv-acc.json \
 -v ~/Desktop/options/conf.yaml:/options/conf.yaml \
-my-image --conf ./options/conf.yaml
+my-image servicedirectory --conf ./options/conf.yaml
 ```
+
+## With Cloud Map
+
+Using Cloud Map is pretty trivial, as it just requires following this
+[example](./usage.md#with-cloud-map) and properly mounting the credentials
+file. You can follow the example above and modify it accordingly.
```

Comment:
Regarding the credentials file, I assume the reader looks for the `default` profile. I don't have a default profile, because I want to be forced to explicitly chose the profile (and thus the account I want to use) on the command line every time I run the AWS CLI. This is in order to avoid surprises due to assumptions on which profile I think is active. I have seen this setup recommended somewhere and it makes sense to me.

All this to say that we should either allow the user to specify the profile, or have an error message saying that we **require** the default profile. I prefer the first solution, but I can live with the second too. (When debugging credential issues, it is very helpful to have good error messages).

Reply:
Actually, the easiest solution to this is to document that we can pass an environment variable to docker specifying the profile using ` -e AWS_PROFILE=my_profile` on the docker command line

Reply:
I have added a note on how to change the profile, thanks!